### DISCUSSION TITLE
BBOS Release v0.3

### DISCUSSION THREAD

**Discussion by mcknly** [mcknly/breadboard-os#31] [Announcements]
The [3rd release](https://github.com/mcknly/breadboard-os/tree/v0.3) of BreadboardOS has a number of exciting new features! These include:
- #21 - Fix for the onboard LED on Pico W boards (thanks @Kintar)
- #26 - RP2350 support
- Command history! ([mcknly/microshell #1](https://github.com/mcknly/microshell/pull/1) - thanks to @RoStCode's [fork](https://github.com/RoStCode/microshell/tree/issue6_cmd_history) of microshell)
- #27 - Complete abstraction of platform-specific references from top-level configuration files

Thanks to the community and keep those contributions coming!!

**Comment by kintar** [comment]
Awesome! I might even get time to contribute again come the new year. >.<
